updating/index.adoc

```diff
@@ -0,0 +1,90 @@
+
+[id='updating-clusters-overview-understand-update-service']
+== Understanding the OpenShift Update Service
+xref:../updating/understanding-the-update-service.adoc#understanding-the-update-service[Understanding the OpenShift Update Service]: For clusters with internet accessibility, Red Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public APIs. For more information, check the following information:
+
+* xref:../updating/understanding-the-update-service.html#unmanaged-operators_understanding-the-update-service[Understanding support policy for unmanaged Operators]
+
+['updating-clusters-overview-install-and-configure-update-service']
+== Installing and configuring the OpenShift Update Service
+xref:../updating/installing-update-service.adoc#installing-update-service[Installing and configuring the OpenShift Update Service]:  For clusters with internet accessibility, Red Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public APIs. However, clusters in a restricted network have no way to access public APIs for update information. For more information, check the following actions:
+
+* xref:../updating/installing-update-service.adoc#update-service-install[Installing the OpenShift Update Service Operator]
+* xref:../updating/installing-update-service.adoc#update-service-delete-service[Uninstalling the OpenShift Update Service Operator]
+* xref:../updating/installing-update-service.adoc#update-service-create-service[Creating an OpenShift Update Service application]
+* xref:../updating/installing-update-service.adoc#update-service-uninstall[Deleting an OpenShift Update Service application]
+
+['updating-clusters-overview-upgrade-channels-and-releases']
+== Upgrading channels and releases
+xref:../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels_understanding-upgrade-channels-releases[Upgrading channels and releases]: Upgrade channels allow you to choose an upgrade strategy. Upgrade channels are connected to a minor version of {product-title}. Upgrade channels control only release selection and do not impact the version of the cluster that you install; the openshift-install binary file for a specific version of {product-title} always installs that version. If you do not want the Cluster Version Operator to fetch available updates from the upgrade recommendation service, you can use the `oc adm upgrade channel` command in the `oc` to configure an empty channel. For more information, check the following information:
```

Comment:
I see we use `the upgrade recommendation service` here and some other wheres. But we call it `The OpenShift Update Service (OSUS)`. We'd better to unify the name to reduce the misunderstanding.

Reply:
Addressed

Reply:
ack